CruzHacks/cruzhacks-2021-website
# Fix sponsors logo visibility on mobile.

Problem
=======
The SVG logos with embedded images are not showing correctly on mobile. 

Solution
========
Add back the width properties for the image tags. 

Steps to Verify:
-----------------------
- Checkout the `chore/fix-sponsors-svg-on-ios` branch.
- Verify that all logos are visible and in the correct place! **I've only tested on a few devices**.

Screenshots (optional):
-----------------------
Before:
<img src="https://user-images.githubusercontent.com/20177171/103918139-4025dc80-50c3-11eb-8cf4-f194fe5e638e.png" width="500px">

After:
<img src="https://user-images.githubusercontent.com/20177171/103918157-474cea80-50c3-11eb-834d-657ec4ceae0a.png" width="500px">
